### PR TITLE
Remove Shipping calculator settings and link to the WooCommerce admin settings

### LINKED ISSUE #11010: Remove Shipping calculator settings and link to the WooCommerce admin settings
Context pdFofs-1qj-p2

## Problem

image
image

## Solution
Keep merchant shipping settings in the admin only and:
– Cart block: update the Settings tab for the Shipping inner block as follows: remove shipping calculator toggle and make the block use the WC settings
– Checkout block: update the Shipping options inner block settings as follows:
remove the Calculations toggle and make the block use the WC Admin setting
– both areas will display instead something [similar to this area](https://woocheckout.files.wordpress.com/2023/09/image-1.png?w=582), but with the appropriate wording and link

### PR DESCRIPTION
**Pull request by mikejolley**
## What

Removes block level settings that enable/disable the shipping calculator and whether costs are displayed in favour of a link to the main settings page. 

This PR also clears up the verbiage used in other areas to keep text consistent. This includes:

- Inconsistent use of "the woocommerce settings" vs "store settings"
- "options" vs "settings". Since "options" conflicted with shipping options (rates) I avoided the term completely

See screenshots for the new panels.

This does introduce the side effect of "losing" the block level setting, but we cannot really sync this back up because it would affect other blocks and the classic checkout. Should only be a minor inconvenience.

Fixes #11010

## Why

See https://github.com/woocommerce/woocommerce-blocks/issues/11010

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to a page containing the Cart Block
2. Highlight the shipping costs row.
3. You should see a panel called "Shipping Calculations" with a link to the shipping options page
4. Go to a page containing the Checkout block
5. Highlight the shipping methods block (the shipping/local pickup toggle).
6. You should only see the shipping methods panel here which links to zones/local pickup
7. Highlight the shipping options block.
8. You should see a panel called "Shipping Calculations" with a link to the shipping options page

* [ ] Do not include in the Testing Notes 
* [ ] Should be tested by the development team exclusively 

## Screenshots or screencast

Cart:
Screenshot 2023-10-09 at 14 16 29

Checkout Shipping Methods:
Screenshot 2023-10-09 at 14 18 30

Checkout Shipping Options:
Screenshot 2023-10-09 at 14 18 44

## WooCommerce Visibility

Required:

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental
* [ ] N/A

## Checklist

Required:
* [ ] This PR has either a `[type]` label or a `[skip-changelog]` label.
* [ ] This PR is assigned to a milestone.

Conditional:
* [ ] This PR has a changelog description (if `[skip-changelog]` label is not present).
* [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR adds/removes an experimental interfaces, and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
* [ ] This PR has been accessibility tested.
* [ ] This PR has had any necessary documentation added/updated.

## Changelog

> Removed block level shipping calculator and hide costs option in favour of the global setting

### CODE REVIEW

File: assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/attributes.tsx [hunk 1]
```diff
@@ -1,13 +1,4 @@
-/**
- * External dependencies
- */
-import { getSetting } from '@woocommerce/settings';
-
 export default {
```
Comment (opr) [inline assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/attributes.tsx]: Can we move this to `block.json` now that we're not using dynamic values for any attributes? If not in this PR we could make a follow up issue to do this?

File: assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/block.tsx [hunk 2]
```diff
@@ -24,7 +19,10 @@ const Block = ( {
 	return (
 		<TotalsWrapper className={ className }>
 			<TotalsShipping
-				showCalculator={ isShippingCalculatorEnabled }
+				showCalculator={ getSetting(
```
Comment (opr) [inline assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/block.tsx]: Applying this suggestion prevents a TS error

```suggestion
				showCalculator={ getSetting< boolean >(
```

**Review by opr — CHANGES_REQUESTED** [review]
Changes look great, there's one test failing because it tries to toggle the calculator:

`shows empty cart when changing the view and allows toggling of shipping calculator `

We can just remove the part where it selects the shipping block and keep the empty cart check, like so:

```patch
diff --git a/tests/e2e/tests/cart/cart-block.merchant.block_theme.spec.ts b/tests/e2e/tests/cart/cart-block.merchant.block_theme.spec.ts
index 76fb7a316..bb5b08558 100644
--- a/tests/e2e/tests/cart/cart-block.merchant.block_theme.spec.ts
+++ b/tests/e2e/tests/cart/cart-block.merchant.block_theme.spec.ts
@@ -188,24 +188,6 @@ test.describe( 'Merchant → Cart', () => {
                                                ' [data-type="woocommerce/empty-cart-block"]'
                                )
                        ).toBeHidden();
-
-                       await editor.selectBlocks(
-                               await editorUtils.getBlockByName(
-                                       'woocommerce/cart-order-summary-shipping-block'
-                               )
-                       );
-                       await editor.openDocumentSettingsSidebar();
-                       const shippingLabel = editorUtils.page.getByLabel(
-                               'Shipping calculator'
-                       );
-                       await shippingLabel.check();
-                       await expect(
-                               editor.canvas.getByText( 'Change address' )
-                       ).toBeVisible();
-                       await shippingLabel.uncheck();
-                       await expect(
-                               editor.canvas.getByText( 'Change address' )
-                       ).toBeHidden();
                } );
        } );
 } );
```

**Comment by mikejolley** [comment]
Thanks @opr passing now. Can you approve?

**Review by opr — APPROVED** [review]
Thanks Mike 🥂

**Pull request merged**